test/cutensor/elementwise_binary.jl

```diff
@@ -130,12 +130,16 @@ eltypes = ((Float16, Float16),
         @test hD ≈ permutedims(A, p) .- C
 
         α = rand(eltyD)
-        ctD = LinearAlgebra.axpy!(α, ctA, ctC)
+        ctD = CuTensor(copy(ctC.data), copy(ctC.inds))
+        ctE = LinearAlgebra.axpy!(α, ctA, ctD)
+        @test ctE == ctD
         hD = collect(ctD.data)
         @test hD ≈ α.*permutedims(A, p) .+ C
 
         γ = rand(eltyD)
-        ctD = LinearAlgebra.axpby!(α, ctA, γ, ctC)
+        ctD = CuTensor(copy(ctC.data), copy(ctC.inds))
```

Comment:
Same here.

Reply:
We can get away without making a copy here, because this is the last test. But I still think it should be done to avoid future headaches and for consistency. What do you think?